test/react-checkbox-group_spec.js

```diff
@@ -26,6 +26,21 @@ describe('ReactCheckboxGroup', function() {
     expect(boxes.length).to.equal(2);
   });
 
+  it('Checks the correct boxes when `disabled` is used', function() {
```

Comment:
Thanks for adding the test @forum-is ! I would simply change the name of the test to "Disables a box when `disabled` is used"